src/main/java/com/google/swarm/tokenization/common/AuditInspectDataTransform.java

```diff
@@ -59,7 +59,7 @@ public Row apply(Row input) {
                   input.getRow("value").getInt64("total_bytes_inspected").longValue(),
                   Util.INSPECTED)
               .build();
-      LOG.info("Audit Row {}", aggrRow.toString());
+      LOG.info("FileTrackerTransform:MergePartialStatsRow: Audit Row {}", aggrRow.toString());
```

Comment:
I think this is AuditInspectDataTransform

Reply:
Yes the class name is AuditInspectDataTransform. But for logging I have maintained the usage of Transformation names that will be displayed on UI DAG. So here the name of this Transformation is FileTrackerTransform and the sub step is MergePartialStatsRow. Can take out MergePartialStatsRow if it doesn't seem to be right as a log message.

Reply:
ok makes sense.